packages/core/ui/editable-text-base/index.ios.ts

```diff
@@ -189,6 +190,17 @@ export abstract class EditableTextBase extends EditableTextBaseCommon {
 
 		this.nativeTextViewProtected.autocorrectionType = newValue;
 	}
+	public setSelection(start: number, stop?: number) {
+		const view = this.nativeTextViewProtected;
+		if (stop !== undefined) {
+			const begin = view.beginningOfDocument;
+			view.selectedTextRange = view.textRangeFromPositionToPosition(view.positionFromPositionOffset(begin, start), view.positionFromPositionOffset(begin, stop));
```

Comment:
Perhaps this makes this line a bit more readable
```suggestion
			view.selectedTextRange = view.textRangeFromPositionToPosition(
						view.positionFromPositionOffset(begin, start),
						view.positionFromPositionOffset(begin, stop)
			);
```